app/workers/asset_manager_attachment_redirect_url_update_worker.rb

```diff
@@ -0,0 +1,20 @@
+class AssetManagerAttachmentRedirectUrlUpdateWorker < WorkerBase
+  def perform(attachment_data_id)
+    attachment_data = AttachmentData.find(attachment_data_id)
```

Comment:
Did you make a conscious decision to use `AttachmentData.find(attachment_data_id)` vs `AttachmentData.find_by(id: attachment_data_id)` here? This looks inconsistent with [`AssetManagerAttachmentReplacementIdUpdateWorker#perform`][1].

If we leave it like this and the AttachmentData is not found, an `ActiveRecord::RecordNotFound` exception will be raised and the job will be retried.

I don't know what the right approach is, but we should standardise on a single approach for all these attachment-related workers and if we adopt the approach in this commit, the choice should be explained in the commit note.

[1]: https://github.com/alphagov/whitehall/blob/bda339b6bd2b11a1f642780c6ed2772a5d675d41/app/workers/asset_manager_attachment_replacement_id_update_worker.rb#L3

Reply:
Good point. I've addressed this in c28675113.